### PR TITLE
py-mupdf: Submission

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.5.1 21G83 x86_64
Xcode 13.4.1 13F100

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-mupdf/Portfile
Comment (reneeotten) [inline python/py-mupdf/Portfile:50]: this is the default set by the "python" PG so can/should be removed.

**Review by reneeotten — CHANGES_REQUESTED** [review]
from the CI log you'll see it cannot run "swig", likely a missing dependency. Are you positive this worked locally in trace-mode? One small comment besides that the Portfile looks good to me!

**Comment by mascguy** [comment]
Rebased against master, to pick up `mupdf` update.

**Comment by mascguy** [comment]
While CI is succeeding, this cannot be merged until `mupdf` is fixed. See comment: https://github.com/macports/macports-ports/pull/15970#issuecomment-1238475692

**Comment by essandess** [comment]
Rebased.

**Comment by mascguy** [comment]
Just a friendly reminder that the following needs to be fixed, before this can be merged:
* [65799 - mupdf @1.20.3: builds fail across-the-board: expected ';' after top level declarator](https://trac.macports.org/ticket/65799)

**Comment by essandess** [comment]
> Just a friendly reminder that the following needs to be fixed, before this can be merged:
> 
> * [65799 - mupdf @1.20.3: builds fail across-the-board: expected ';' after top level declarator](https://trac.macports.org/ticket/65799)

I don't know how to fix that, and don't understand the issue. Builds locally across the board for me, and also here in the CI.

**Comment by mascguy** [comment]
> > Just a friendly reminder that the following needs to be fixed, before this can be merged:
> > 
> > * [65799 - mupdf @1.20.3: builds fail across-the-board: expected ';' after top level declarator](https://trac.macports.org/ticket/65799)
> 
> I don't know how to fix that, and don't understand the issue. Builds locally across the board for me, and also here in the CI.

See my latest update on the ticket. But in short, we need to disable optimization, when compiling generated resources.

**Comment by mascguy** [comment]
> > I don't know how to fix that, and don't understand the issue. Builds locally across the board for me, and also here in the CI.
> 
> See my latest update on the ticket. But in short, we need to disable optimization, when compiling generated resources.

Unfortunately that doesn't fix the issue, so I'm not sure where to go from here. And since `mupdf` fails across-the-board on the buildbots, this port is blocked until the issue has been fixed.

Since you updated the port, perhaps you can jump in and help try to figure this out? Please?  :-)

**Comment by reneeotten** [comment]
Closed by dadf9ffdd85684357d2aa94250683fa108533d91

**Pull request closed without merging**
